app/routes/messaging.py

```diff
@@ -48,15 +54,24 @@ async def send_trust_ping(
 ):
     """
     Trust ping
```

Comment:
This one and previous one can use a custom summary

And for "trust ping", I'm wondering what is a trust ping? Is it just pinging a connection to indicate it's still active?